redirects-transition.conf

```diff
@@ -1072,7 +1076,20 @@ rewrite ^/pubrec/cfsdd/cfsdd.shtml https://www.fec.gov/introduction-campaign-fin
 rewrite ^/pubrec/cfsdd/cfsded_000.pdf https://www.fec.gov/resources/cms-content/documents/cfsded.pdf redirect;
 
 # rad/ redirects
-rewrite ^/rad/index.shtml /rad/index.html redirect; 
+rewrite ^/rad/BitcoinFAQ.shtml https://www.fec.gov/help-candidates-and-committees/filing-reports/bitcoin-contributions/ redirect;
+rewrite ^/rad/candidates/FEC-ReportsAnalysisDivision-CandidateCommittees.shtml https://www.fec.gov/help-candidates-and-committees/guides/?tab=candidates-and-their-authorized-committees redirect;
+rewrite ^/rad/documents/best_practices_to_avoid_pitfalls_2018.pdf https://www.fec.gov/resources/cms-content/documents/best_practices_to_avoid_pitfalls_2018.pdf redirect;
+rewrite ^/rad/documents/RADProcessDocument3-16-16.pdf https://www.fec.gov/resources/cms-content/documents/RAD_Review_Procedures_Feb2018.pdf redirect;
+rewrite ^/rad/FEC-ReportsAnalysisDivision-Penalties.shtml https://www.fec.gov/legal-resources/enforcement/ redirect;
+rewrite ^/rad/FederalElectionCommission-RAD-ElectronicFilingPasswords.shtml https://www.fec.gov/help-candidates-and-committees/filing-reports/electronic-filing/ redirect;
+rewrite ^/rad/FederalElectionCommission-RAD-RespondingtoRFAIs.shtml https://www.fec.gov/help-candidates-and-committees/request-additional-information/ redirect;
+rewrite ^/rad/index.shtml /rad/index.html redirect;
+rewrite ^/rad/index.html https://www.fec.gov/help-candidates-and-committees/ redirect;
+rewrite ^/rad/JointFundraisingCommittees.shtml https://www.fec.gov/help-candidates-and-committees/making-disbursements/joint-fundraising-candidates-political-committees redirect;
```

Comment:
This should have a / at the end of the page to redirect to, like this: ‘joint-fundraising-candidates-political-committees/‘

Reply:
Updated in the second commit to the branch. Since you didn't comment on the redirects to anchor links that don't end in a "/" I didn't update those rows. If you think those should be updated, I'll go in and do that, let me know.

For example this url has no slash at the very end: "https://www.fec.gov/help-candidates-and-committees/guides/?tab=political-party-committees"